### PR TITLE
missed blocks

### LINKED ISSUE #387: Use history API for Missed blocks
We can calculate if a BP missed a round or blocks using state history API which is more acurate

### LINKED ISSUE #443: Handle error when history API is down
The `hapi` service has an unhandled error when starting up with the history service API down 

```
events.js:292
      throw er; // Unhandled 'error' event
      ^
Error: Unexpected server response: 404
    at ClientRequest.<anonymous> (/app/node_modules/ws/lib/websocket.js:604:7)
    at ClientRequest.emit (events.js:315:20)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:634:27)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:117:17)
    at Socket.socketOnData (_http_client.js:503:22)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:302:12)
    at readableAddChunk (_stream_readable.js:278:9)
    at Socket.Readable.push (_stream_readable.js:217:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:186:23)
Emitted 'error' event on WebSocket instance at:
    at abortHandshake (/app/node_modules/ws/lib/websocket.js:722:15)
    at ClientRequest.<anonymous> (/app/node_modules/ws/lib/websocket.js:604:7)
    [... lines matching original stack trace ...]
    at Socket.Readable.push (_stream_readable.js:217:10)
```

### PR DESCRIPTION
**Pull request by adriexnet**
### What does this PR do?

- Resolve #387 
- Resolve #443  

### Steps to test

1. make run laccahin
2. navigate to http://localhost:3000/missed-blocks

#### CheckList

- [x] Follow proper Markdown format
- [x] The content is adequate
- [x] The content is available in both english and spanish
- [x] I Ran a spell check

### CODE REVIEW

File: .github/workflows/push-lacchain-environment.yaml [hunk 1]
```diff
@@ -116,9 +116,11 @@ jobs:
           HAPI_HASURA_URL: 'http://monitor-hasura:8080/v1/graphql'
```
Comment (xavier506) [inline .github/workflows/push-lacchain-environment.yaml:116]: @adriexnet  can we please restore the value for `HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT: ''` in the PR.

File: hapi/src/services/missed-blocks.service.js [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+      }
+    }  
+  `
+  const data = await hasuraUtil.request(query)
+
+  return data?.schedule_history?.length > 0 ? data.schedule_history[0] : {}
+}
+
+const getProducersBySchedule = async () => {
+  // TODO: validate if it's normal that the first block
```
Comment (xavier506) [inline hapi/src/services/missed-blocks.service.js:20]: this is a good question for @elmato

**Review by xavier506 — APPROVED** [review]
Amazing work @adriexnet 🥇

File: hapi/src/services/missed-blocks.service.js [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+  // was generated by a producer with empty name
```
Comment (xavier506) [inline hapi/src/services/missed-blocks.service.js:21]: this is a good question for @elmato

File: .github/workflows/push-lacchain-environment.yaml (hunk 1 above)
Comment (adriexnet) [reply .github/workflows/push-lacchain-environment.yaml:116]: sure, I will do it

Comment (adriexnet) [reply .github/workflows/push-lacchain-environment.yaml:116]: done

File: .github/workflows/push-lacchain-environment.yaml [hunk 4]
```diff
@@ -112,7 +112,7 @@ jobs:
           HAPI_EOS_MECHANICS_PASSWORD: ${{ secrets.HAPI_EOS_MECHANICS_PASSWORD }}
           HAPI_EOS_WALLET_URL: http://monitor-wallet:8888
           HAPI_HASURA_ADMIN_SECRET: ${{ secrets.HAPI_HASURA_ADMIN_SECRET }}
-          HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT: ''
+          HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT: 'ws://observer.eosio.cr:8888'
```
Comment (xavier506) [inline .github/workflows/push-lacchain-environment.yaml:115]: 👍

File: webapp/src/routes/Home.js [hunk 5]
```diff
@@ -0,0 +1,220 @@
+/* eslint camelcase: 0 */
```
Comment (tetogomez) [inline webapp/src/routes/Home.js:1]: Why add the home file again?

we moved this file to `webapp/src/routes/Home/index.js`

please check it!

**Review by tetogomez — CHANGES_REQUESTED** [review]
Please see the comments

**Pull request merged**